components/perl/Clone/perl-integrate-module.conf

```diff
@@ -0,0 +1,5 @@
+%include-3%
```

Comment:
This file lacks the license header.  I believe all files, especially manually created, should have the license header to avoid confusion.